### PR TITLE
Refactoring

### PR DESCRIPTION
**Pull request by JakeChampion**
Refactor suggestions from @Integralist to make it easer to read, understand & DRY.

### CODE REVIEW

File: lib/npmrc-switcher.js [hunk 1]
```diff
@@ -1,55 +1,61 @@
...
-    shell('/bin/cp', [source, target], function(){
-        callback();
-    });
+function copySourceToDestination(source, destination) {
+    shell('/bin/cp', [source, destination]);
 }
 
-function returnToDefault() {
-    renameFile(npmrcUserBackupPath, npmrcUserPath)
+function returnToBackedUpFile() {
```
Comment (JakeChampion) [inline lib/npmrc-switcher.js:45]: @Integralist Do you think this is descriptive enough about what is going on inside the function?

File: lib/npmrc-switcher.js [hunk 2]
```diff
@@ -1,55 +1,61 @@
...
+    renameFile(baseBackup, baseFile);
 }
 
-function backupCurrent() {
-    renameFile(npmrcUserPath, npmrcUserBackupPath)
+function backupFileCurrentlyInUse() {
```
Comment (JakeChampion) [inline lib/npmrc-switcher.js:49]: @Integralist Same question here.

**Comment by Integralist** [comment]
Seems OK to me

**Comment by JakeChampion** [comment]
@Integralist Think this is in a merge-able state now?

**Comment by Integralist** [comment]
Just change double " to single ' (you do this in your require calls but maybe else where too)

**Comment by JakeChampion** [comment]
Changed

**Comment by Integralist** [comment]
Merge away!

**Pull request merged**